frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1901,7 +1544,34 @@ const ManageHostsPage = ({
                 )}
             </div>
           </div>
-          {renderActiveFilterBlock()}
+          {/* TODO: look at improving the props API for this component. Im thinking
+          some of the props can be defined inside HostsFilterBlock */}
+          <HostsFilterBlock
+            policyResponse={policyResponse}
+            selectedLabel={selectedLabel}
+            policyId={policyId}
+            policy={policy}
+            macSettingsStatus={macSettingsStatus}
+            softwareId={softwareId}
+            mdmId={mdmId}
+            mdmEnrollmentStatus={mdmEnrollmentStatus}
+            lowDiskSpaceHosts={lowDiskSpaceHosts}
+            osId={osId}
+            osName={osName}
+            osVersion={osVersion}
+            osVersions={osVersions}
+            munkiIssueId={munkiIssueId}
+            munkiIssueDetails={munkiIssueDetails}
+            isOnlyObserver={isOnlyObserver}
+            softwareDetails={softwareDetails}
+            mdmSolutionDetails={mdmSolutionDetails}
+            handleClearRouteParam={handleClearRouteParam}
+            handleClearFilter={handleClearFilter}
+            onChangePoliciesFilter={handleChangePoliciesFilter}
+            onChangeMacSettingsFilter={handleMacSettingsStatusDropdownChange}
+            onClickEditLabel={onEditLabelClick}
+            onClickDeleteLabel={toggleDeleteLabelModal}
+          />
```

Comment:
It could be nice to collapse some of these props into an object type that is a subset of `ILoadHostOptions`  
```
         <HostsFilterBlock
            params={
              softwareId,
              mdmId,
              ...
            }
            ...
         />
```